examples/text_labels_and_annotations/rainbow_text.py

```diff
@@ -62,11 +63,17 @@ def rainbow_text(x, y, strings, colors, orientation='horizontal',
 
         # Need to draw to update the text position.
         text.draw(canvas.get_renderer())
+        # Get window extent
         ex = text.get_window_extent()
+        # Convert window extent from pixels to inches
+        ex = fig.dpi_scale_trans.inverted().transform_bbox(ex)
```

Comment:
The first comment is kind of redundant; can the second comment be updated to say _why_ that is being done?

Reply:
I added a small general explanation. If you were looking for something more specific, let me know! I don't mind adjusting it further.

Reply:
LGTM